perspective_reddit_bot/README.md

```diff
@@ -0,0 +1,44 @@
+# Perspective Moderation Bot
+
+## Description
+
+This repository contains a python script for a Reddit bot which uses the Perspective API to help moderate comments on a subreddit. By specifying rules in the rules.yaml file, the bot will check new comments on a given subreddit for triggering these rules and can report those that do. In order for the actions to be applied, the reddit credentials provided to the bot must have moderation permissions on the subreddit being moderated.
+
+## Setup
+
+1.  Setup a (virtualenv)[https://virtualenvwrapper.readthedocs.io/en/latest/] for
+    the project (recommended, but technically optional).
+
+    Python 2:
```

Comment:
I think the code probably works with both 2 and 3, right? Or is this just about how to get virtualenv working on Python 2? Do you know if it's different for 3?